go/test/endtoend/transaction/twopc/twopc_test.go

```diff
@@ -671,6 +691,10 @@ func TestDTResolveAfterRMPrepare(t *testing.T) {
 	require.NoError(t, err)
 	_, err = conn.Execute(qCtx, "insert into twopc_user(id, name) values(8,'bar')", nil)
 	require.NoError(t, err)
+	// Also do an insertion into a table that has a consistent lookup vindex.
```

Comment:
the comment says insertion, but the query is an update. I'm confused

Reply:
I shall clarify the confusion by changing the comment to say update!